MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -83,7 +106,7 @@ public int GetHashCode(char b)
         /// <remarks>Can be used to calculate character bounds when implementing custom SpriteFont rendering.</remarks>
         public Dictionary<char, Glyph> GetGlyphs()
```

Comment:
For now I could generate the Dictionary<char, Glyph> on the fly in GetGlyphs(). `_glyphsDictionary` is not used anywhere else.